plugin/adsk/scripts/mayaUsdMenu.mel

```diff
@@ -154,6 +241,116 @@ global proc mayaUsdMenu_createMenuCallback() {
     }
 }
 
+///////////////////////////////////////////////////////////////////////////////
+// mayaUsd_updateSelectionKindMenuItem
+// Update the menuItem radio button in the Selection Mode menu
+global proc mayaUsd_updateSelectionKindMenuItem() {
+    string $selectionKind = mayaUSD_getSelectionKind();
+    switch($selectionKind) {
+        case "":
+            menuItem -e -radioButton on selectKindNone;
+            break;
+        case "model":
+            menuItem -e -radioButton on selectKindModel;
+            break;
+        case "group":
+            menuItem -e -radioButton on selectKindGroup;
+            break;
+        case "assembly":
+            menuItem -e -radioButton on selectKindAssembly;
+            break;
+        case "component":
+            menuItem -e -radioButton on selectKindComponent;
+            break;
+        case "subcomponent":
+            menuItem -e -radioButton on selectKindSubcomponent;
+            break;
```

Comment:
Do we need special cases for all these known types? Why not just let the loop below (lines 273-279) handle all the types? The only special case is the empty one.

Reply:
Now that the build-in will not have a capital letter I will only use a special case for the empty one.